Dockerfile

```diff
@@ -5,19 +5,20 @@ FROM $DEPENDENCY_IMAGE AS builder
 COPY . ./
 
 RUN  \
-    python ./build_with_conan.py --release --parallel 4\
+    python3 ./build_with_conan.py --release --parallel 4\
     && cp build/silo_test . \
     && cp build/siloApi .
 
 
-FROM alpine:3.20 AS server
+FROM ubuntu:22.04 AS server
 
 WORKDIR /app
 COPY docker_default_preprocessing_config.yaml ./default_preprocessing_config.yaml
 COPY docker_runtime_config.yaml ./runtime_config.yaml
 COPY --from=builder /src/siloApi ./
 
-RUN apk update && apk add onetbb=2021.12.0-r0 curl jq
+RUN apt update \
```

Comment:
Also running `apt upgrade` would also be good to update the installed stuff.

Reply:
Hmm, but isn't this precisely contrary to your previously mentioned point of reproducible builds?

Not that I am against this, but if we do want the upgrade, maybe we should rethink our decision on fixed versions afterall?

Reply:
This is also about general security upgrades for installed system dependencies that SILO doesn't use (e.g. ssl)